Storage/rest/basic.h

```diff
@@ -174,7 +175,7 @@ struct Basic {
   };
 
   // LCOV_EXCL_START
-  static Response ErrorMethodNotAllowed() {
+  static Response ErrorMethodNotAllowed(const std::string&) {
     return Response("Method not allowed.\n", HTTPResponseCode.MethodNotAllowed);
```

Comment:
Perhaps `Method "$METHOD" not allowed.` ?
And the '`\n'` is not required here any longer, right?